PowerValidatedSolutions.psm1

```diff
@@ -7536,18 +7535,18 @@ Function Undo-vRLIDeployment {
                                         Start-Sleep 10
                                         Watch-vRSLCMRequest -vmid $($newRequest.requestId)
                                         if (!(Get-vRSLCMEnvironment | Where-Object {$_.environmentName -eq $environmentName})) {
-                                            Write-Output "Removal of vRealize Log Insight from vRealize Suite Lifecyle Manager ($($vcfVrslcmDetails.fqdn)): SUCCESSFUL"
+                                            Write-Output "Removal of Aria Operations for Logs from Aria Suite Lifecycle ($($vcfVrslcmDetails.fqdn)): SUCCESSFUL"
                                         } else {
-                                            Write-Error "Removal of vRealize Log Insight from vRealize Suite Lifecyle Manager ($($vcfVrslcmDetails.fqdn)): POST_VALIDATION_FAILED"
+                                            Write-Error "Removal of Aria Operations for Logs from Aria Suite Lifecycle ($($vcfVrslcmDetails.fqdn)): POST_VALIDATION_FAILED"
                                         }
                                     } else {
-                                        Write-Output "Removal request of vRealize Log Insight Submitted Successfully (Request Ref: $($newRequest.requestId))"
+                                        Write-Output "Removal request of Aria Operations for Logs Submitted Successfully (Request Ref: $($newRequest.requestId))"
                                     }
                                 } else {
-                                    Write-Error "Removel request of vRealize Log Insight failed, check the vRealize Suite Lifecycle Manager UI: POST_VALIDATION_FAILED"
+                                    Write-Error "Removel request of Aria Operations for Logs failed, check the Aria Suite Lifecycle UI: POST_VALIDATION_FAILED"
```

Comment:
```suggestion
                                    Write-Error "Removal request of Aria Operations for Logs failed, check the Aria Suite Lifecycle UI: POST_VALIDATION_FAILED"
```

Reply:
Nice catch! Resolved in force push to https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/80a44deb6323815e007842449672e8aaacdce83c.